### PR TITLE
Add unsubscribe user modal in admin

### PR DESCRIPTION
**Pull request by ranquild**
A premium feature to unsubscribe a user from all dashboard subscriptions and alerts and delete ones the user has created. 

How to test:
- Run `yarn dev-ee`
- Create a question with an alert
- Navigate to `Admin > People`
- In the context menu for a user select `Unsubscribe from all subscriptions / alerts`
- `Confirm` unsubscribing
- Check that the alert was archived, e.g. by navigating to the question or to `Account settings > Notifications`

Screen Shot 2021-09-15 at 1 54 08 pm
Screen Shot 2021-09-15 at 1 59 23 pm

Please note that the corresponding cypress test fails as the BE endpoint does not delete slack subscriptions; to be fixed separately.

### CODE REVIEW

File: enterprise/frontend/src/metabase-enterprise/plugins.js [hunk 1]
```diff
@@ -5,9 +5,8 @@ import MetabaseSettings from "metabase/lib/settings";
 // NOTE: temporarily use "latest" for Enterprise Edition docs
 MetabaseSettings.docsTag = () => "latest";
 MetabaseSettings.isEnterprise = () => true;
-// PLUGINS:
 
-// import "./management";
+// PLUGINS:
```
Comment (alxnddr) [inline enterprise/frontend/src/metabase-enterprise/plugins.js:9]: @ranquild should this be a part of `advanced_config`? It is not mentioned in Cam's message and the doc about new feature flags. @brunobergher is it correct that we are adding notifications & alerts management as a part of `audit-app`? It looks like this small feature does not belong to audit but seems related

Comment (brunobergher) [reply enterprise/frontend/src/metabase-enterprise/plugins.js:9]: This is part of `audit-app` for two reasons:
- It's a small item on its own, but part of the bigger subscription audit UI (which hasn't merged yet)
- Cam is weary of adding things behind a new flag this close to shipping

**Review by alxnddr — COMMENTED** [review]
Codewise looks great 👍  The only thing is changing the enterprise plugin from `advanced_config` to the existing `audit-app`

File: enterprise/frontend/src/metabase-enterprise/plugins.js (hunk 1 above)
Comment (ranquild) [reply enterprise/frontend/src/metabase-enterprise/plugins.js:9]: There is a message from @camsaul 10 hours ago about moving the BE handler behind `advanced_config` feature flag. What should I do?

**Comment by ranquild** [comment]
Blocked until there is a new endpoint available.

File: enterprise/frontend/src/metabase-enterprise/plugins.js (hunk 1 above)
Comment (ranquild) [reply enterprise/frontend/src/metabase-enterprise/plugins.js:9]: Fixed

**Comment by ranquild** [comment]
> Codewise looks great 👍 The only thing is changing the enterprise plugin from `advanced_config` to the existing `audit-app`

Fixed

**Comment by ranquild** [comment]
@alxnddr The PR is ready for review; the Cypress test passes only with BE changes from https://github.com/metabase/metabase/pull/17981.

**Review by alxnddr — APPROVED** [review]
👍

**Pull request merged**